python/composio/cli/add.py

```diff
@@ -336,11 +336,6 @@ def _handle_basic_auth(
         force_new_integration=True,
         labels=labels,
     )
```

Comment:
Consider adding a comment here explaining that user access data is already saved during connection initialization to prevent future reintroduction of duplicate calls.